packages/dev-scripts/examples/util.ts

```diff
@@ -114,7 +116,7 @@ export type Files = Record<
 
 export function getProjectFiles(project: Project): Files {
   const dir = path.resolve("../../", project.pathFromRoot);
-  const files = glob.globSync(dir + "/**/*", {
+  const files = glob.globSync(glob.win32.convertPathToPattern(dir + "/**/*"), {
```

Comment:
Now we're also calling win32.convertPathToPattern on mac / unix right? Does that make sense? It seems a bit weird to me!